src/App.css

```diff
@@ -36,3 +36,15 @@
     transform: rotate(360deg);
   }
 }
+
+body {
+  overflow: hidden;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'PT Sans', sans-serif;
```

Comment:
have the decency to read through the code the rest of the group put together before adding styles the fuck with the whole website.